### PR TITLE
Manolo tlc day 7 3c-3e

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3D.txt [hunk 1]
```diff
@@ -558,17 +557,17 @@ But seeing how upset she was, I feel like I'm the one at fault... Completely and
...
 -- Page 272, Offset 17235.
 -- だから待て、止まれってばっ！」
-So just stop. Stop where you are!"
+Wait, come on, stop already!"
 }
 [04dac99d46c40ac4a55ca0d8039d84a2fa826ba0]{
 -- Page 273, Offset 17236.
 -- あわてて白い背中を追いかける。
-Flustered, I run after her white back.
+Flustered, I chase after her white back.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3D.txt:565]: if you're using 'white back' here you might as well use it in the other files for consistency tbh.

**Review by Avuxo — COMMENTED** [review]
great work as always!
I'll finish up with the last file tomorrow

File: script/Arcueid/Day 7/07_00_ARC07_3D.txt (hunk 1 above)
Comment (nolo16) [reply script/Arcueid/Day 7/07_00_ARC07_3D.txt:565]: the other instance of 白い背中 (arc_07_1_2) from the files i worked on so far was also translated as 'white back', will take note for the future though

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 2]
```diff
@@ -373,7 +372,7 @@ That's a shame. The fact that you can see death even here."
 [bd0e2fdb261433a87918317c20ab124415acf14d]{
 -- Page 301, Offset 17313.
 -- 命が有る無しの問題さえ無関係。動いているもの、破壊できるものなら例外なく停止させる―――なによ。ほんとの化物はあなたの方じゃない」
-Even a question of whether a life exists or not is irrelevant. If it moves and can be destroyed, then you can stop it, no exception―――Dear me. The one that's truly a monster is you."
+Even a question of whether life exists or not is irrelevant. If it moves and can be destroyed, then you can end it, no exception―――Dear me. The one that's truly a monster is you."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:375]: ―――Man. the true monster here is you.

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 3]
```diff
@@ -388,17 +387,17 @@ Even a question of whether a life exists or not is irrelevant. If it moves and c
 [f5a3c6162601239e485964eb636ec553553f92a6]{
 -- Page 302, Offset 17316.
 -- 死という負債を、他人の血で誤魔化し続ける吸血鬼だから」
-It is a vampire that continues to bypass the debt of death with the blood of others."
+It's a vampire that continues to cheat the debt called death with the blood of others."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:390]: It's a vampire that continues to cheat that debt we call death with the blood of others.

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 4]
```diff
@@ -388,17 +387,17 @@ Even a question of whether a life exists or not is irrelevant. If it moves and c
...
 -- Page 303, Offset 17317.
 -- アルクェイドは歩を速める。
-Arceuid fastens her pace.
+Arceuid quickens her pace.
 }
 [bcd8e747166f7ca25695bef9a8e4687f4442126e]{
 -- Page 303, Offset 17318.
 -- 一直線に、どこにでもいるような男に向かって。
-She heads in a straight line toward the man who seemed to be everywhere.
+She heads in a straight line toward the commonplace man.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:400]: run-of-the-mill man

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 5]
```diff
@@ -448,17 +447,17 @@ I can hear the awfully close sound of my heartbeat.
 [f2a6f8eff67ff4795fa16c2f4e96f012dfccc26b]{
 -- Page 308, Offset 17328.
 -- まだ夜も深くない時間。
-It is hardly late in the night.
+It's hardly late into the night.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:450]: It's hardly late yet.

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 6]
```diff
@@ -448,17 +447,17 @@ I can hear the awfully close sound of my heartbeat.
...
 -- Page 308, Offset 17329.
 -- 通り過ぎていく車のヘッドライト。
-The headlight of the passing car.
+The headlights of the passing car.
 }
 [dc267ff12179141e800a4b9e3c0f116d97eff58f]{
 -- Page 308, Offset 17330.
 -- 頭上を過ぎていく電車の通過音。
-The sound of the train raging overhead.
+Everything abruptly came to a halt.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:460]: was this bad copy-paste? revert

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 7]
```diff
@@ -478,12 +477,12 @@ My glasses―――I need to put on my glasses.
...
 -- Page 309, Offset 17334.
 -- そうしないと、厭なものを見てしまう。
-Otherwise I will see something unpleasant.
+Otherwise I'll see something unpleasant.
 }
 [c6562d5c55ea6658720f63fee4a5a789610bfefd]{
 -- Page 309, Offset 17335.
 -- 今まで視えていたものなんて入り口にすぎないぐらいの淵を、見る事になってしまう。
-If I don't, it would be like staring straight into the abyss, with everything I've seen up until this point being nothing but its mere doorway.
+If I don't, it'd be like staring straight into the abyss, with everything I've seen up until this point being no more thits mere doorway.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:485]: ? revert

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 8]
```diff
@@ -498,7 +497,7 @@ But my body won't move.
 [9c2127fe49a56acd7b1a5aec3fa60c0ed375e766]{
 -- Page 310, Offset 17338.
 -- ツギハギだらけのセカイを視る裸眼は、魅了されたように、アルクェイドが入っていった<黒洞|こくどう>を、闇越しに見つめている。
-My naked eyes, which had seen the spattered imperfections of this world, looked upon the dark road in which Arcueid had entered in awe.
+My naked eyes, which had seen the patchwork of imperfections that riddled this world, looked upon the dark void which Arcueid had entered in awe.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:500]: I caution against adapting ツギハギ blanketly as patchwork but this change is great. Also "riddle"

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 9]
```diff
@@ -533,8 +532,8 @@ Suddenly, everything came to a halt.
 [c56712b1773646096b98c44b8edf51b9c6f99823]{
 -- Page 312, Offset 17345.
 -- 絶対零度の月の下。
-// ?????
-Below the absolute zero temperatures of the moon.
+// ????? Another ???? from me
+Beneath the absolute zero temperatures of the moon.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:536]: Nasu sure fucking loves 絶対零度 that piece of fuck doesn't even know what he's doing to us, does he?
anyays, drop "temperatures" but "beneath" is good

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 10]
```diff
@@ -609,7 +608,7 @@ Holding back the nausea gradually rising up to my throat, I put on my glasses wi
 [291c661544e741133c2a0a0a43a4980812f9cd3b]{
 -- Page 317, Offset 17360.
 -- 音と光が戻る。
-The light and sound goes back to normal.
+The light and sound returns to normal.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:611]: return

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 11]
```diff
@@ -659,7 +658,7 @@ Arcueid comes out of the darkness, limping forward and more out of breath than I
 [8a698a2fc0056f4eaec2eef0d89031f4b8d4f936]{
 -- Page 321, Offset 17370.
 -- 気まずそうに顔を背けて、ゆらりと、アルクェイドは俺の横をすり抜けていく。
-Turning away her beaten face, Arcueid slips through my side with a swaying motion.
+Turning away her battered face, Arcueid slips through my side with a swaying motion.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:661]: slips past me

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 12]
```diff
@@ -669,12 +668,12 @@ Her manner of walking was reminiscient of a sick patient.
...
 -- Page 322, Offset 17372.
 -- ―――正直、こっちだって吐き気が治まっていない。
-―――To be honest, I still haven't gotten over my nausea on my end.
+―――To be honest, I still haven't gotten over the nausea on my end.
 }
 [5e9d56c00292058db18e5c4856ba9384149045b8]{
 -- Page 322, Offset 17373.
 -- けど、あんなに苦しげなアルクェイドを前にしたら、そんな事はどうでもよくなった。
-But after seeing the painful look of Arcueid in front of me, those kind of things seemed to have taken care of itself.
+But after seeing the painful look of Arcueid in front of me, those kind of things were of little importance.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:676]: kinds of things

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 13]
```diff
@@ -684,7 +683,7 @@ But after seeing the painful look of Arcueid in front of me, those kind of thing
 [ffb01e270472d8e7fb4d373fa9668e28add7ab75]{
 -- Page 323, Offset 17375.
 -- 立ち去っていこうとするアルクェイドの肩を掴む。
-I grab a hold of the shoulder of Arcueid who planned on departing.
+I grab hold of Arcueid shoulder as she attempts to leave.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:686]: Arcueid's

**Review by Avuxo — APPROVED** [review]
some more changes but great work!

**Pull request merged**